src/index.ts

```diff
@@ -6,3 +6,4 @@ SPDX-License-Identifier: Apache-2.0
 export * from './AwsSolutions/aws-solutions';
 export * from './NIST-800-53/nist-800-53';
 export * from './common';
+export * from './NIST-800-53/nist-800-53';
```

Comment:
Duplicate export should be removed
```suggestion
export * from './NIST-800-53/nist-800-53';
```